docs/swagger2.yml

```diff
@@ -27,119 +27,133 @@ consumes:
 produces:
   - application/json
 paths:
-  /funcs:
+  /apps/{app}/fns:
     get:
-      operationId: "ListFuncs"
-      summary: "Get all funcs."
-      description: "Get a list of all the funcs in alphabetical order."
+      operationId: "ListFns"
+      summary: "Get all fns"
+      description: "Get a list of all the Functions in alphabetical order."
       tags:
-        - Funcs
+        - Fns
       parameters:
+        - $ref: '#/definitions/appIDParam'
         - $ref: '#/definitions/cursorParam'
         - $ref: '#/definitions/perPageParam'
       responses:
         200:
-          description: List of funcs.
+          description: List of fns.
           schema:
-            $ref: '#/definitions/FuncsWrapper'
+            $ref: '#/definitions/FnsWrapper'
+        default:
+          description: Unexpected error
+          schema:
+            $ref: '#/definitions/Error'
+    post:
```

Comment:
was hoping to avoid post semantics, and this mostly hinges on using ids as the primary identifier I understand. 2 threads open for this, I'll post in the other one on id vs name.